etcd/etcd2-live-http-to-https-migration.md

```diff
@@ -0,0 +1,192 @@
+# etcd2 Live HTTP to HTTPS migration
+
+If you already have existing etcd2 cluster which uses insecure HTTP connections and you would like to switch from HTTP to HTTPS and in addition enable TLS certificate authentication, you have to follow instruction below.
+
+## Prepare Cluster Components
+
+### Check insecure port availability
+
+By default fleet and flannel use two etcd client ports: 2379 and 4001. Last one is used for etcd v0.4 backward compatibility. In this example we will use 4001 TCP port for insecure HTTP connections and our etcd members will listen that port only on local interface for security reason. This will allow us to switch transparently to HTTPS configuration without services downtime and simplify configuration
```

Comment:
I'm also rethinking if we should strictly advise listening on localhost only. We tell people to run dedicated etcd clusters, which means they need to be listening on a different interface. Should we just add some language that states how to choose your interface for your needs?